commands/mmctl_test.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/mattermost/mattermost-server/v5/api4"
 )
 
+var EnableEnterpriseTests string
```

Comment:
TIL that we can't set `bool` value with `-ldflags`.

Reply:
:facepalm: (TIL as well)

Reply:
dafuk